extras/ams_wrapper/src/api/models/vehicle_detection_adas_model.py

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import sys
+import json
+from logger import get_logger
+from api.models.model import Model
+from api.types import Tag, Rectangle, SingleEntity, Entity
+
+logger = get_logger(__name__)
+
+
+class VehicleDetectionAdas(Model):
+
+    def load_default_labels(self):
+        labels_path = os.path.abspath(__file__).replace(".py",".json")
+        try:                                                                          
+            with open(labels_path, 'r') as labels_file:                               
+                data = json.load(labels_file)                                         
+                self.labels = data
+        except Exception as e:                                                        
```

Comment:
I think there are two possible ways to fail here. One is when program is unable to open file, second one is when JSON is not valid and unable to load.

Reply:
This will be changed once we switch to model templates from your PR.
We will add error codes for this situation.